ansible-ipi-install/inventory/hosts.sample

```diff
@@ -169,6 +169,9 @@ pullsecret=""
 # (Optional) Change the boot mode of the OpenShift cluster nodes to legacy mode (BIOS). Default is UEFI.
 #bootmode=legacy
 
+# (Optional) A MAC address to use for the external NIC on the bootstrap VM. This is optional and if blank is generated by libvirt.
+#externalMACAddress="52:54:00:86:af:07"
```

Comment:
I wonder if we should refrain from entering an example here
to avoid people simply commenting it and clashing with each other (if using the same lab)

eg. how about ? 
#externalMACAddress="52:54:00:XX:XX:XX"

Reply:
Thanks @derekhiggins for the input. I replaced it in the doc.